src/main/java/net/conjur/api/Endpoints.java

```diff
@@ -14,6 +15,8 @@
     private static final String ACCOUNT_PROPERTY_NAME = "CONJUR_ACCOUNT";
     private static final String AUTHN_URL_PROPERTY_NAME = "CONJUR_AUTHN_URL";
```

Comment:
Since this is used in multiple locations, I think it's better to have these constants in a central location. WDYT?

Reply:
Done